eZ/Publish/API/Repository/Tests/ContentTypeServiceTest.php

```diff
@@ -1416,6 +1416,43 @@ public function testUpdateContentTypeDraftWithNewTranslation()
         );
     }
 
+    /**
+     * @covers \eZ\Publish\API\Repository\ContentTypeService::updateContentTypeDraft
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testUpdateContentTypeWithDescriptionClearing()
+    {
+        $repository = $this->getRepository();
+        $contentTypeService = $repository->getContentTypeService();
+
+        $contentTypeDraft = $this->createContentTypeDraft();
+        $contentTypeService->publishContentTypeDraft($contentTypeDraft);
+
+        $contentType = $contentTypeService->loadContentType($contentTypeDraft->id);
+        // sanity check
+        self::assertEquals(
+            ['eng-US', 'ger-DE'],
+            array_keys($contentType->getNames())
+        );
+
+        $contentTypeDraft = $contentTypeService->createContentTypeDraft($contentType);
+        $updateStruct = $contentTypeService->newContentTypeUpdateStruct();
+        $updateStruct->languageCode = 'ger-DE';
+        $updateStruct->descriptions = null;
```

Comment:
There's a bit of an architectural issue here as well. First of all, the PHPDoc allows arrays only. Second, `null` indicates that an element of a struct is unset and is to be untouched. However in current implementation it depends on a language code. It's possible it behaved similarly prior the changes here, but still - it's inconsistent with respect to the rest of the APIs.

```php
        $updateStruct->descriptions = [];
```
works as well currently, so I'd stick with that at least (but it depends on the direction taken, mentioned earlier)